### PR TITLE
Remove komoju gem

### PR DESCRIPTION
**Pull request by k2nr**
Please don't merge

### CODE REVIEW

**Comment by k2nr** [comment]
tests all pass

File: lib/active_merchant/billing/gateways/komoju.rb [hunk 1]
```diff
@@ -89,6 +84,25 @@ def commit(params)
                      :authorization => success ? response["id"] : nil)
       end
 
+      def post_data(params)
```
Comment (rramsden) [inline lib/active_merchant/billing/gateways/komoju.rb]: Could we use a JSON string as the post body instead? Not sure if we need to convert a hash to a querystring, easier to use `to_json` for this, no?

Comment (k2nr) [inline lib/active_merchant/billing/gateways/komoju.rb]: ah yes, we can use JSON as request body. I'll change it.

**Comment by shioyama** [comment]
This looks good to me aside from @rramsden's comment :+1:

**Comment by rramsden** [comment]
Looks good to me, nice work :+1:

**Comment by k2nr** [comment]
cherry-picked the commits. closing

**Pull request closed without merging**